### PR TITLE
Use Crypo framework from soffes repo

### PR DESCRIPTION
**Pull request by ps2**

### CODE REVIEW

**Comment by loudnate** [comment]
Yep, that's exactly what I was going to do :)

File: Cartfile [hunk 1]
```diff
@@ -1 +1 @@
-github "loudnate/Crypto" "master"
+github "soffes/Crypto" "master"
```
Comment (loudnate) [inline Cartfile]: s/`"master"`/`~> 0.4.0` would probably be better.

**Comment by ps2** [comment]
And travis CI is happy!

**Comment by ps2** [comment]
Thanks @beached and @mhaeberli for looking into this.

**Comment by loudnate** [comment]
LGTM

**Pull request merged**